lib/index.ts

```diff
@@ -13,6 +13,9 @@ export { filter, load, save, loadFromText, add, addExclude, create };
 
 const debug = newDebug('snyk:policy');
 
+/** Returns the version of the latest policy schema */
+export const latestVersion = () => 'v1.25.1'; // only major _should_ matter, but deferring for now
```

Comment:
Happy to see the "schema version" decoupled from the "library version". Note that any changes here result in big downstream changes and should be considered carefully